### PR TITLE
Lunch register [WIP]

### PR DESCRIPTION
**Pull request by bmarkons**

### CODE REVIEW

File: scripts/lunch.coffee [hunk 1]
```diff
@@ -28,11 +28,94 @@ location= '45.257217,19.845845'
 https = require 'https'
 
 module.exports = (robot) ->
+
+  robot.brain.data.lunch = {}
+
+  is_brain_empty = () ->
+    for x,y of robot.brain.data.lunch
+      return false
```
Comment (shiroyasha) [inline scripts/lunch.coffee]: This method looks a bit funky? Do you want to test if the object is empty?

Comment (shiroyasha) [inline scripts/lunch.coffee]: You can probably use `Object.keys(myObject).length == 0` instead.

File: scripts/lunch.coffee [hunk 2]
```diff
@@ -28,11 +28,94 @@ location= '45.257217,19.845845'
...
+  location_exists = (location) ->
+    location of robot.brain.data.lunch
+
+  lunch_list = () ->
+    console.log robot.brain.data.lunch
+    for location, users of robot.brain.data.lunch
+      users_for_location = ""
+      for user, i in users
+        users_for_location += user.name
+        users_for_location += "\n"
```
Comment (shiroyasha) [inline scripts/lunch.coffee]: You can try to use `users.map((el) => "#{u.name}\n").join("")`

File: scripts/lunch.coffee [hunk 3]
```diff
@@ -28,11 +28,94 @@ location= '45.257217,19.845845'
...
+      title: location
+      value: users_for_location
+      short: false
+
+  add_user_on_location = (msg) ->
+    location = msg.match[1]
+    if(!location_exists(location))
```
Comment (shiroyasha) [inline scripts/lunch.coffee]: We prefer `unless` instead of `if !something`

File: scripts/lunch.coffee (hunk 1 above)
Comment (bmarkons) [inline scripts/lunch.coffee]: yes! Thanks :)

File: scripts/lunch.coffee (hunk 3 above)
Comment (bmarkons) [inline scripts/lunch.coffee]: I noticed that :) Is there some specific reason for that?

Comment (shiroyasha) [inline scripts/lunch.coffee]: The biggest reason is readability. You can maybe take a look at this too https://github.com/bbatsov/ruby-style-guide. (I know it is ruby specific, but many things apply to coffescript as well).

**Pull request closed without merging**